app/build.gradle

```diff
@@ -53,6 +53,14 @@ dependencies {
     kapt 'com.github.bumptech.glide:compiler:4.9.0'
     implementation 'com.github.bumptech.glide:glide:4.9.0'
 
+    // DI
+    // Koin for Android
+    implementation "org.koin:koin-android:2.0.1"
+    // or Koin for Lifecycle scoping
+    implementation "org.koin:koin-androidx-scope:2.0.1"
+    // or Koin for Android Architecture ViewModel
+    implementation "org.koin:koin-androidx-viewmodel:2.0.1"
```

Comment:
Follow the file convention and use single quotes.

Reply:
Indeed. A good next task in this project is to add ktlint. 
I'll try to do this in my free time.

https://ktlint.github.io/